api/groups/addressGroup.go

```diff
@@ -215,6 +222,29 @@ func (ag *addressGroup) getUsername(c *gin.Context) {
 	shared.RespondWithSuccess(c, gin.H{"username": userName, "blockInfo": blockInfo})
 }
 
+// getCodeHash returns the code hash for the address parameter
+func (ag *addressGroup) getCodeHash(c *gin.Context) {
+	addr := c.Param("address")
+	if addr == "" {
+		shared.RespondWithValidationError(c, errors.ErrGetUsername, errors.ErrEmptyAddress)
```

Comment:
wrong error var returned, please create another one

Reply:
Yes, you are right. Line 241, I also used:

```
shared.RespondWithInternalError(c, errors.ErrCouldNotGetAccount, err)
```

And I believe it is not the correct error too, so I changed it to be:

```
shared.RespondWithInternalError(c, errors.ErrGetCodeHash, err)
```